ccl/multiregionccl: TestMultiRegionDataDriven_regional_by_table failed

ccl/multiregionccl.TestMultiRegionDataDriven_regional_by_table [failed](https://mesolite.cluster.engflow.com/invocations/default/ef2acb24-760e-47bd-8a8b-c996537e0b5c?testReportRun=1&testReportShard=4&testReportAttempt=1#targets-Ly9wa2cvY2NsL211bHRpcmVnaW9uY2NsOm11bHRpcmVnaW9uY2NsX3Rlc3Q=) on master @ [33c53dd564a8f103082604e71b0dd5500f260db9](https://github.com/cockroachdb/cockroach/commits/33c53dd564a8f103082604e71b0dd5500f260db9):


```
        /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:179:
          exec-sql idx=0
          ALTER TABLE db.rbt SET LOCALITY REGIONAL BY TABLE IN "us-east-1";
          ----
    datadriven_test.go:151: 
        /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:183:
          exec-sql idx=0
          ALTER DATABASE db DROP REGION "us-west-1"
          ----
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 10.003439818s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 20.004321118s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 30.004705917s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 40.004845008s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 50.014558019s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 1m0.015058131s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 1m10.029548709s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 1m20.029678327s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 1m30.032872362s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 1m40.034547323s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 1m50.035555352s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 2m0.03714017s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 2m10.037331475s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 2m20.037537959s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 2m30.039053885s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 2m40.063190314s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 2m50.063466845s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 3m0.068719588s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 3m10.06905471s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 3m20.077105676s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 3m30.079389952s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 3m40.092699105s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 3m50.092953878s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 4m0.093276135s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 4m10.093407261s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 4m20.099727899s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 4m30.099911467s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 4m40.101284067s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 4m50.101603321s
    datadriven.go:357: /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 5m0.101809633s
    datadriven_test.go:151: 
        /mnt/engflow/worker/work/3/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187:
          wait-for-zone-config-changes db-name=db table-name=rbt num-voters=3 num-non-voters=3 leaseholder=0 voter=1,2 non-voter=6,7,8 not-present=3,4,5
        
        expected:
        
        found:
          expected node 4 to not be present but had replica type non-voter
    panic.go:615: -- test log scope end --
test logs left over in: outputs.zip/logTestMultiRegionDataDriven_regional_by_table2675951510
--- FAIL: TestMultiRegionDataDriven_regional_by_table (427.99s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=1</code>
 - <code>shard=4</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMultiRegionDataDriven_regional_by_table.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>


Jira issue: CRDB-58523

ccl/multiregionccl.TestMultiRegionDataDriven_regional_by_table [failed](https://mesolite.cluster.engflow.com/invocations/default/3e25d8f9-0bd3-404a-885a-526fcbf5a2fd?testReportRun=2&testReportShard=4&testReportAttempt=1#targets-Ly9wa2cvY2NsL211bHRpcmVnaW9uY2NsOm11bHRpcmVnaW9uY2NsX3Rlc3Q=) on master @ [cbc6c60a25f6b2d8ebe89ebd182e2b7afa419287](https://github.com/cockroachdb/cockroach/commits/cbc6c60a25f6b2d8ebe89ebd182e2b7afa419287):


```
    datadriven_test.go:151: 
        /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187:
          wait-for-zone-config-changes db-name=db table-name=rbt num-voters=3 num-non-voters=3 leaseholder=0 voter=1,2 non-voter=6,7,8 not-present=3,4,5
          ----
    datadriven_test.go:151: 
        /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:194:
          exec-sql idx=0
          ALTER DATABASE db SURVIVE REGION FAILURE
          ----
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 10.000563961s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 20.000901265s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 30.007447544s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 40.00845378s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 50.011095895s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 1m0.01143407s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 1m10.016378178s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 1m20.0168148s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 1m30.016924689s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 1m40.018157206s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 1m50.020068254s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 2m0.031595015s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 2m10.032332708s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 2m20.041195327s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 2m30.042253552s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 2m40.055491137s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 2m50.05579478s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 3m0.061473336s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 3m10.062738543s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 3m20.062930986s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 3m30.063061524s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 3m40.0634122s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 3m50.063537764s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 4m0.071530881s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 4m10.071701037s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 4m20.073959706s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 4m30.07409818s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 4m40.07796726s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 4m50.078718756s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200: still running after 5m0.084336271s
    datadriven_test.go:151: 
        /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:200:
          wait-for-zone-config-changes db-name=db table-name=rbt num-voters=5 num-non-voters=0 voter=6,7,8
        
        expected:
        
        found:
          expected node 6 to have replica type voter but was not found
    panic.go:615: -- test log scope end --
test logs left over in: outputs.zip/logTestMultiRegionDataDriven_regional_by_table2335673262
--- FAIL: TestMultiRegionDataDriven_regional_by_table (430.30s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=2</code>
 - <code>shard=4</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMultiRegionDataDriven_regional_by_table.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

ccl/multiregionccl.TestMultiRegionDataDriven_regional_by_table [failed](https://mesolite.cluster.engflow.com/invocations/default/efba721b-da2d-4fb6-9601-ed980685b19d?testReportRun=2&testReportShard=4&testReportAttempt=1#targets-Ly9wa2cvY2NsL211bHRpcmVnaW9uY2NsOm11bHRpcmVnaW9uY2NsX3Rlc3Q=) on master @ [94a1e22436ad935d8c7f9524cb8fc0fd7b24faaa](https://github.com/cockroachdb/cockroach/commits/94a1e22436ad935d8c7f9524cb8fc0fd7b24faaa):


```
        /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:179:
          exec-sql idx=0
          ALTER TABLE db.rbt SET LOCALITY REGIONAL BY TABLE IN "us-east-1";
          ----
    datadriven_test.go:151: 
        /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:183:
          exec-sql idx=0
          ALTER DATABASE db DROP REGION "us-west-1"
          ----
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 10.000011682s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 20.000594136s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 30.001107151s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 40.001223701s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 50.00362543s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 1m0.017947374s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 1m10.019730917s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 1m20.019993783s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 1m30.02012499s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 1m40.020327812s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 1m50.020466422s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 2m0.020844707s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 2m10.021083171s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 2m20.024738812s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 2m30.024876378s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 2m40.037914432s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 2m50.038922637s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 3m0.039161176s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 3m10.047717194s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 3m20.048043487s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 3m30.048473771s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 3m40.059211672s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 3m50.062142058s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 4m0.063264165s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 4m10.064301958s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 4m20.064557152s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 4m30.064779972s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 4m40.065052574s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 4m50.065235935s
    datadriven.go:357: /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187: still running after 5m0.065399581s
    datadriven_test.go:151: 
        /mnt/engflow/worker/work/2/exec/bazel-out/k8-fastbuild/bin/pkg/ccl/multiregionccl/multiregionccl_test_/multiregionccl_test.runfiles/com_github_cockroachdb_cockroach/pkg/ccl/multiregionccl/testdata/regional_by_table:187:
          wait-for-zone-config-changes db-name=db table-name=rbt num-voters=3 num-non-voters=3 leaseholder=0 voter=1,2 non-voter=6,7,8 not-present=3,4,5
        
        expected:
        
        found:
          0 of 4 live stores are able to take a new replica for the range (2 already have a voter, 1 already has a non-voter); replicas must match constraints [{+region=eu-central-1:1} {+region=eu-west-1:1} {+region=us-central-1:1} {+region=us-east-1:1}]; voting replicas must match voter_constraints [{+region=us-east-1}]
    panic.go:615: -- test log scope end --
test logs left over in: outputs.zip/logTestMultiRegionDataDriven_regional_by_table2762723970
--- FAIL: TestMultiRegionDataDriven_regional_by_table (439.68s)
```

Parameters:
 - <code>attempt=1</code>
 - <code>run=2</code>
 - <code>shard=4</code>
<details><summary>Help</summary>
<p>

See also: [How To Investigate a Go Test Failure \(internal\)](https://cockroachlabs.atlassian.net/l/c/HgfXfJgM)
</p>
</details>
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*TestMultiRegionDataDriven_regional_by_table.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/bazci/githubpost/issues)

</sub>

--- a/pkg/ccl/multiregionccl/datadriven_test.go
+++ b/pkg/ccl/multiregionccl/datadriven_test.go
@@ -59,7 +59,6 @@ func TestMultiRegionDataDriven_regional_by_row(t *testing.T) {
 func TestMultiRegionDataDriven_regional_by_table(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
-	skip.WithIssue(t, 160803)
 	testMultiRegionDataDriven(t, "regional_by_table")
 }
 

--- a/pkg/cmd/roachtest/tests/acceptance.go
+++ b/pkg/cmd/roachtest/tests/acceptance.go
@@ -116,7 +116,6 @@ func registerAcceptance(r registry.Registry) {
 			{
 				name:     "mismatched-locality",
 				fn:       runMismatchedLocalityTest,
-				skip:     "TODO(#160845): unskip this",
 				numNodes: 3,
 			},
 		},

